desktop/src/app.rs

```diff
@@ -137,15 +136,10 @@ impl App {
                     // Don't render when minimized to avoid potential swap chain errors in `wgpu`.
                     if !minimized {
                         if let Some(mut player) = self.player.get() {
-                            player.render();
-                            let renderer = player
-                                .renderer_mut()
-                                .downcast_mut::<WgpuRenderBackend<MovieView>>()
-                                .expect("Renderer must be correct type");
-                            self.gui
-                                .lock()
-                                .expect("Gui lock")
-                                .render(Some(renderer.target()));
+                            if player.is_playing() {
+                                player.render();
```

Comment:
to be sure, we don't need to keep refreshing the render in case something draws on top of it?

Reply:
We render ruffle to a texture, which shouldn't be able to change if the player is frozen. That texture is always rendered to the surface, and then UI after it, so it doesn't need any extra attention